littlebot_control/launch/spawn_diffdriver_controller.launch.py

```diff
@@ -0,0 +1,49 @@
+import os
+from launch_ros.actions import Node
+from launch import LaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.actions import RegisterEventHandler
+from launch.event_handlers import OnProcessExit
+
+
+def generate_launch_description():
+    joint_state_broadcaster_spawner = Node(
+        package="controller_manager",
+        executable="spawner",
+        arguments=["joint_state_broadcaster", "--controller-manager", "/controller_manager"],
+    )
+
+    robot_controller_spawner = Node(
+        package="controller_manager",
+        executable="spawner",
+        arguments=["littlebot_velocity_controller", "--controller-manager", "/controller_manager"],
+    )
+
+    rviz_node = Node(
+        package="rviz2",
+        executable="rviz2",
+        name="rviz2",
+        output="log",
+        # arguments=["-d", rviz_config_file],
+    )
+
+    delay_robot_controller_spawner_after_joint_state_broadcaster_spawner = RegisterEventHandler(
+        event_handler=OnProcessExit(
+            target_action=joint_state_broadcaster_spawner,
+            on_exit=[robot_controller_spawner],
+        )
+    )
```

Comment:
line 30 too long (93 > 79 characters)

```suggestion
    delay_robot_controller_spawner_after_joint_state_broadcaster_spawner =\
        RegisterEventHandler(
            event_handler=OnProcessExit(
                target_action=joint_state_broadcaster_spawner,
                on_exit=[robot_controller_spawner],
            )
        )
```

Reply:
fixed [here](https://github.com/NestorDP/littlebot/pull/10/commits/33db77a6b46130d550d8f299364a59b44734b9e4)